CMakeLists.txt

```diff
@@ -196,6 +196,10 @@ if(ANDROID)
   set(DISABLE_ZERO_TIER ON)
   set(TTF_FONT_DIR \"\")
 
+  if(BINARY_RELEASE OR CMAKE_BUILD_TYPE STREQUAL "Release")
+    set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3 -flto=full")

```

Comment:
Without this we get an error stating: `Optimization level must be between 0 and 3` when doing a release build.

I don't know if this is relevant for other targets or if there is a better way to specify it :shrug:

Reply:
Maybe [add_compile_options()](https://cmake.org/cmake/help/latest/command/add_compile_options.html#command:add_compile_options)?

Also, it might be about time to add an `android_defs.cmake` for all these custom settings.

Reply:
Interesting, where did you find the `-flto=full` thing? Most compilers these days do thin LTO.

Reply:
https://stackoverflow.com/questions/56238019/linker-error-when-enabling-link-time-optimization-in-ndk

Reply:
Let's add a comment with a link to the NDK issue? https://github.com/android/ndk/issues/721